initials.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
```

Comment:
Awesome! Your code works and has all the needed requirements. The only thing I might add is your name at the top.